tests/shopify_python/test_google_styleguide.py

```diff
@@ -113,3 +113,26 @@ def test_catch_blank_passes(self):
         """)
         with self.assertAddsMessages():
             self.walk(root)
+
+    def test_try_exc_fnly_complexity(self):
+        root = astroid.builder.parse("""
+        try:
+            a = [x for x in range(0, 10) if x < 5]
+            a = sum((x * 2 for x in a)) + (a ** 2)
+        except AssertionError as assert_error:
+            if set(assert_error).startswith('Division by zero'):
+                raise MyException(assert_error, [x for x in range(0, 10) if x < 5])
+            else:
+                a = [x for x in range(0, 10) if x < 5]
+                raise
+        finally:
+            a = [x for x in range(0, 10) if x < 5]
+            a = sum((x * 2 for x in a))
+        """)
+        with self.assertAddsMessages(
+            pylint.testutils.Message('finally-too-long', node=root.body[0], args={'found': 24}),
+            pylint.testutils.Message('try-too-long', node=root.body[0].body[0], args={'found': 28}),
+            pylint.testutils.Message('except-too-long', node=root.body[0].body[0].handlers[0],
```

Comment:
Could you name these nodes something more meaningful, so the reader can know what `root.body[0].body[0].handlers[0]` refers to without parsing the tree?